Makefile

```diff
@@ -551,7 +599,7 @@ ci-job: deps-update go-generate generate fmt vet lint shellcheck bashate fmt tes
 
 .PHONY: clean
 clean:
-	-rm $(LOCALBIN)/$(BINARY_NAME)
+	rm -rf $(LOCALBIN)
```

Comment:
This is one spot where it feels dangerous to me. I wonder if this should have an if/else so that it only removes the dir if `$(LOCALBIN)` is equal to `$(PROJECTDIR)/bin`, and just gives a warning otherwise.

Reply:
what about if I individually removed the binaries inside the folder instead?
e.g.
```
rm $(YQ)
rm $(YAMLLINT)
rm $(OPM)
... etc
```